odoo/fields.py

```diff
@@ -1154,9 +1153,22 @@ class Integer(Field):
     """ Encapsulates an :class:`int`. """
     type = 'integer'
     column_type = ('int4', 'int4')
+    bigint = False
 
     group_operator = 'sum'
 
+    def __init__(self, string=Default, **kwargs):
+        super(Integer, self).__init__(string=string, **kwargs)
+        if kwargs.get('bigint'):
+            self.column_type = ('int8', 'int8')
+            self.column_cast_from = ('int4',)
```

Comment:
alternative: define `column_type` and `column_cast_from` as properties:
```suggestion
    @property
    def column_type(self):
        return ('int8', 'int8') if self.bigint else ('int4', 'int4')
```

Reply:
Thanks, suggestion taken